src/components/organism/CreatePool/index.tsx

```diff
@@ -0,0 +1,338 @@
+          selectDisabled={true}
+        />
+        <TokenAmountInput
+          tokenText={selectedTokenB?.tokenSymbol}
+          labelText={t("tokenAmountInput.youPay")}
+          tokenIcon={<DotToken />}
+          tokenValue={selectedTokenAssetValue?.tokenValue}
+          onClick={() => setIsModalOpen(true)}
+          onSetTokenValue={(value) => setSelectedTokenBValue(value)}
+        />
+        <div className="mt-1 text-small">{transferGasFeesMessage}</div>
+
+        <div className="flex w-full flex-col gap-2 rounded-lg bg-purple-50 px-4 py-6">
+          <div className="flex w-full justify-between text-medium font-normal text-gray-200">
+            <div className="flex">{t("tokenAmountInput.slippageTolerance")}</div>
+            <span>15%</span>
+          </div>
+          <div className="flex w-full gap-2">
+            <div className="flex w-full basis-8/12 rounded-xl bg-white p-1 text-large font-normal text-gray-400">
+              <button
+                className={classNames("flex basis-1/2 justify-center rounded-lg px-4 py-3", {
+                  "bg-white": !slippageAuto,
+                  "bg-purple-100": slippageAuto,
+                })}
+                onClick={() => {
+                  setSlippageAuto(true);
+                  setSlippageValue(15);
+                }}
+              >
+                {t("tokenAmountInput.auto")}
+              </button>
+              <button
+                className={classNames("flex basis-1/2 justify-center rounded-lg px-4 py-3", {
+                  "bg-white": slippageAuto,
+                  "bg-purple-100": !slippageAuto,
+                })}
+                onClick={() => setSlippageAuto(false)}
+              >
+                {t("tokenAmountInput.custom")}
+              </button>
+            </div>
+            <div className="flex basis-1/3">
+              <div className="relative flex">
+                <NumericFormat
+                  value={slippageValue}
+                  onValueChange={({ floatValue }) => setSlippageValue(floatValue)}
+                  fixedDecimalScale={true}
+                  thousandSeparator={false}
+                  allowNegative={false}
+                  className="w-full rounded-lg bg-purple-100 p-2 text-large  text-gray-200 outline-none"
+                  placeholder="15"
+                  disabled={slippageAuto}
+                />
+                <span className="absolute bottom-1/3 right-2 text-medium text-gray-100">%</span>
+              </div>
+            </div>
+          </div>
+
+          {poolExists ? (
+            <div className="flex rounded-lg bg-lime-500 px-4 py-2 text-medium font-normal text-cyan-700">
+              {t("poolsPage.poolExists")}
+            </div>
+          ) : null}
+        </div>
+
+        <Button
+          onClick={handlePool}
+          variant={ButtonVariants.btnInteractivePink}
+          disabled={returnButtonStatus() !== SwapAndPoolStatus.Deposit}
```

Comment:
Are u sure that u need to disable this button if it is not Deposited? Also, it is a not good idea. Now u depend on returnButtonStatus which is a string and SwapAndPoolStatus.Deposit is an enum. returnButtonStatus can be changed tomorrow, for example, I mean the value/string (it can be "Deposited" instead of "Deposit"). That being said u will not get the appropriate value for the comparison